README.md

```diff
@@ -82,6 +82,16 @@ Tags of this image follow Seafile version:
 
   * **SEAFILE_ADMIN_PASSWORD** (required): password for the admin account
 
+  * **SEAFILE_LDAP_URL** (optional): LDAP URL (e.g. ldap://openldap)
```

Comment:
it would be better that the name of these variables does not start with **SEAFILE_**